05/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="cuentos cortos o fabulas con moralejas">
+    <meta name="keyword" content="cuentos, cortos, moralejas, lectura, escritura, aprendizaje">
+    <title>Formularios y Cuentos</title>
+    <meta name="author" content="Jesus David Ceballos Parra">
+  </head>
+  <body>
+    <header>
+      <h1>LAS ZORRA Y LAS UVAS</h1>
+      <p>En pleno Otoño, Cuando las uvas mostraban suculentos ramos, Estaba una zorra tumbada debajo de un arból, Hechando un sueñecito.
+        Justo cuando se despertó, Se fijó en un ramo de uvos que va se veia maduro y le parecio un maravilloso bocado, que no debia dejar escapar.
+        Cuando se incorporó, pudo observar que no seria tarea facíl alcanzar las uvas, Pero no quizo rendirse tan pronto.
+        Comenzó a salta y a saltar, sin obtener ningún éxito.  Decidió acercar algúnas pierdas para elevarse, pero tampoco dio resultado.
+        tanto esfuerzo tenia a la zorra exahusta y, Finalmente, Decidió abandonar el racimo de uvas, pues le parecía imposible alcanzarlo.
+        Cuando ya se marchaba bastante frustada por no haber conseguidos sus uvas
+        Vio a un pajarillo junto al arból de cual colgaba el racimo.
+        La pequeña ave había presenciado toda la escena. La zorra, al darse cuenta de que la habían visto fracasar,se sintió avergonzada y quiso que el pajaro no pensara que no podia conseguir todo lo que ella quisiera, asi que dijo llena de orgullo:
+      </p>
```

Comment:
All of this text in one paragraph? / Todo este texto en un solo p?

Reply:
que xuxa

Reply:
> All of this text in one paragraph? / Todo este texto en un solo p?

yes